sensor/state_of_charge.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+#include "state_of_charge.h"
+
+using namespace std;
+
+vector<int> generateSocValues(int socMaxValue , int socMinValue, int numberOfReadings) {

```

Comment:
Good practice in separating the seperate files(.cpp) for Temperature and State Of Charge.

Reply:
thanks